deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -0,0 +1,95 @@
+# Selectively remove jobs temporarily.
```

Comment:
So the instance group is added, but without any jobs actually running ?

Reply:
Never mind, should have read the ticket. This is about adding the CC job in the api group.

Reply:
The title says "cloud_controller_ng".

Reply:
Huh ... Oh, PR title, right. I was looking at JIRA ticket title, and then description.

Reply:
I'll update JIRA!